src/lib.rs

```diff
@@ -239,6 +239,35 @@ impl From<Hash> for [u8; OUT_LEN] {
     }
 }
 
+/// Transforming `Hash` into `u64` will increase the number of
+/// collisions as `u64` can represent fewer values than `[u8;
+/// OUT_LEN]`.
+///
+/// Bytes from `Hash` are extracted by chunks of 8, and then added to
+/// `0u64` with a modular addition.
+impl From<Hash> for u64 {
+    #[inline]
+    fn from(hash: Hash) -> Self {
+        use core::convert::TryInto;
+
+        let bytes: [u8; OUT_LEN] = hash.into();
+
+        (0..(OUT_LEN / 8))
+            .into_iter()
+            .map(|i| (i * 8, (i + 1) * 8))
+            .fold(0u64, |accumulator, (x, y)| {
+                assert_eq!(y - x, 8);
+
+                accumulator.wrapping_add(u64::from_be_bytes(
+                    (&bytes[x..y])
+                        .try_into()
+                        // SAFETY: OK, because `y - x = 8` and `wrapping_add` expects a `[u8; 8]`.
```

Comment:
Just a terminology quibble: I wouldn't consider a fallible `unwrap`/`expect` to be a safety issue. A correctness issue for sure, but not something that can lead to UB.

Reply:
Correct!